asset/css/search-base.less

```diff
@@ -277,8 +293,11 @@ fieldset:disabled .term-input-area [data-drag-initiator] {
     [data-index] {
       position: relative;
 
+      &:not(:hover) > .remove-action {
```

Comment:
Please move it a few lines down, after or before the definition of `.remove-action` for better readability.